include/seqan3/io/sequence_file/output_format_concept.hpp

```diff
@@ -25,11 +25,40 @@
 namespace seqan3::detail
 {
 
+/*!\brief Internal class used to expose the actual format interface to write sequence records within the file.
```

Comment:
within the file => to/into the file? 
(also in alignment and structure file output concepts)

Reply:
Nice! You also changed the input concepts :)